client/app/visualizations/counter/index.js

```diff
@@ -97,6 +97,10 @@ function CounterRenderer() {
             $scope.stringSuffix = null;
           }
         }
+
+        $timeout(() => {
+          $scope.handleResize();
```

Comment:
Will there be "flickering"?

Reply:
Maybe, just a little bit. On my core i7 I didn't see any flickering. But I cannot change/remove this code - I need to wait for angular to render second line with number, otherwise it computes wrong font size.

Reply:
If it's has `display:none` will we be able to measure it?

Reply:
No.